src/main/java/org/sunbird/core/config/MessageConfig.java

```diff
@@ -0,0 +1,21 @@
+package org.sunbird.core.config;
+
+import org.springframework.context.MessageSource;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.support.ResourceBundleMessageSource;
+import org.sunbird.common.util.Constants;
+
+@Configuration
+public class MessageConfig {
+
+    @Bean
+    public MessageSource messageSource() {
+        ResourceBundleMessageSource messageSource = new ResourceBundleMessageSource();
```

Comment:
resource bundle is not required... 
we are not going to support locale settings from backend services... 
 
just keep these messages in application properties, so that we can configure or change when required...